server.js

```diff
@@ -80,6 +90,8 @@ async function pinningNode () {
         openDB(odbAddress)
       })
     }
+
+    cache.invalidate(address)
```

Comment:
Seems like this is in the wrong place. I think it should be called when we replicate an entry in any db.

Reply:
yeah could be moved, to be more exact, since we know when changes happen. Was just based on quick initial assumption that something MAY have change.

Reply:
actually not going to change until next sprint tests/cleanup. To be more exact, helpful if differentiate between priv vs pub open. And need to fix https://github.com/3box/3box-pinning-server/issues/27 so that its not called over and over. Easy change after, and still valid at the moment, just a bit less effective.